pkgs/development/python-modules/h5netcdf/default.nix

```
@@ -12,14 +12,14 @@
 
 buildPythonPackage rec {
   pname = "h5netcdf";
-  version = "1.4.1";
+  version = "1.5.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.9";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-fIQBq4B/83yXmO3JDZlGdZWJLmxUGl1avrj1OqtTNf4=";
+    hash = "sha256-nWwoFHZ3vQdXh4garbJNL9sLwDGUgby10qVNxLlMS6k=";
   };
 
   nativeBuildInputs = [
```

```suggestion
  build-system = [
```
and use `dependencies`